index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/8b2919a184.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- ------------------The Nav Bar-------------------->
+    <header>
+      <nav>
+        <ul>
+          <li><a href="#">My Logo</a></li>
+          <li>
+            <a href="#"><i class="fa-solid fa-bars"></i></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- -------------------------------The Home Page---------------------------------- -->
+    <section class="main-page">
+      <div class="home-page-details">
+        <article class="section-heading">
+          <h1>I'm Amen</h1>
+          <h2>Glad to see you!</h2>
+        </article>
+        <article class="about-me-text">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </article>
+        <section class="social-links">
```

Comment:
- [ ] I think the article and section tags here are not necessary, kindly take them out for semantic and SEO purposes :recycle: